Minor English changes.

brokep opened the pull request:
File: Hacking.md 
diff:
[https://github.com/dnssec/libdime/commit/f7546c6ecf1085194fdc1ba3ff73c310affb954d](https://github.com/dnssec/libdime/commit/f7546c6ecf1085194fdc1ba3ff73c310affb954d)

Moving from "basic" to "primary" in the description. Rewrote some statements into direct actionable statements.

brokep committed:
Minor English changes. 

Moving from "basic" to "primary" in the description. Rewrote some statements into direct actionable statements.

brokep committed:
Merge pull request #1 from klosnet/develop

Minor English changes.

gabeio reviewed (commented):
I think the markdown was line breaking at ~90-100 characters, this allows for _better_ readability on a terminal window.

gabeio commented on HACKING.md:
```diff
@@ -1,13 +1,10 @@
 
-Right now, the code for the two basic tools (`tools/dime` and
-`tools/signet`) is finished, but they need a lot of testing to be sure they work properly.
+Right now, the code for the two primary tools (`tools/dime` and `tools/signet`) is finished, but they need much testing to be sure they work correctly.
```
`need a lot of testing` is more grammatically correct than `need much testing`

gabeio commented on HACKING.md:
```diff
@@ -37,14 +32,8 @@ tests. Look up their names in the Check documentation:
-https://scan.coverity.com/, a static code analyzer that has found quite
-a few bugs in the code, which need to be fixed. I already fixed some of
-the simple ones, and the remaining ones get more and more trickier.
+If you know how to fix bugs in C, I can give you an account on https://scan.coverity.com/, a static code analyzer that has found quite a few bugs in the code, which need to be fixed. I already fixed some of the simple ones, and the remaining ones get more and trickier.
```
I believe `the remaining ones get more and tricky.` would be the proper way to say this.

ladar committed:
Addressing language changes.

brokep committed:
Create TODO.md: Build, Write, Engage, Drive: 

Creating TODO.md for libdime. Focusing on 6 specific initiatives. Build, Write, Engage, Drive

brokep committed:
Delete TODO.md
